alembic/env.py

```diff
@@ -0,0 +1,84 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from db.models import *
```

Comment:
[clarify] You need to import the entry point for your models, not all of the individual models.